llvm/tools/llvm-snippy/README.md

```diff
@@ -41,11 +41,28 @@ From the ${SNIPPY_PATH} directory, run CMake:
 > cmake --install release/build
 ```
 
-We haven't open-sourced LIT tests yet. When we do that, there will also be instructions here on how to test the generator.
+## Testing
+
+You can test LLVM-snippy by running LIT tests located in `llvm/test/tools/llvm-snippy`. These tests are based on the common for LLVM infrastructure including llvm-lit and FileCheck. For the additional information about LLVM testing infrastructure, please, refer to [LLVM Testing Infrastructure Guide](https://llvm.org/docs/TestingGuide.html).
```

Comment:
These tests are based on the common LLVM infrastructure, including llvm-lit and FileCheck. For additional information about LLVM's testing infrastructure, please refer to the [LLVM Testing Infrastructure Guide].

Reply:
I would omit "please" everywhere in any user-facing doc, there's no need for it.

Reply:
Ok

Reply:
Fixed